cmd/gapit/BUILD.bazel

```diff
@@ -28,6 +28,7 @@ go_library(
         "dump_replay.go",
         "dump_shaders.go",
         "export_replay.go",
+		"graph_visualization.go",
```

Comment:
Whitespace here looks off.

Reply:
I fixed all problems in regard to the format.